### DISCUSSION TITLE
Issues when processing proof of academic status (page reload/camera issues)

### DISCUSSION THREAD

**Discussion by KadenMc** [community/community#67837] [GitHub Education]
### Hi everyone,

I am facing some issues when trying to "Upload proof of your academic status" for the Student Developer Pack.

After successfully using "Upload an image" to upload my proof of enrollment documentation, clicking "Process my application", and waiting for the progress bar to complete, the page will reload with no indication of a submission or any changes at all.

I tried submitting multiple different screenshots in case it isn't accepting my image but is simply failing silently, but it made no difference.

I tried this process both on Chrome and on Microsoft Edge to see if it could be a browser-related problem, but the exact same thing happened.

On another note, I thought maybe the issue could be that I was uploading a picture and not taking one. I tried "Take a picture" on both browsers and was stuck with an infinite loading arrow - the site did not request to use the camera on either browser.

Clearly, something very wrong is going on - perhaps this is what happens if the system is overloaded with requests?

Any insight would be much appreciated!

**Comment by cgorlla** [comment]
Same as #67200
This seems to be an OCR issue where the text cannot be read, resizing the image to make the relevant text larger should work. Wasted an hour on this.

**Reply by mr-september** [reply]
This led me to the solution. Pop the photo into Photoshop, play with Unsharp Mask, Auto Color/Contrast, save (under 1 Mb), and it finally went through. Github of all companies should at least show the user ANY error message.

**Reply by harshil3134** [reply]
yeah the problem was with ocr and it wasn't giving me any error but when I scanned the document through the scanner app and applied the filter it worked on the first try.
